ajax/libs/bootstrap-autohidingnavbar/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "bootstrap-autohidingnavbar",
+  "title": "Bootstrap Auto-Hiding Navbar",
+  "filename": "jquery.bootstrap-autohidingnavbar.min.js",
+  "description": "An extension for Bootstrap's fixed navbar which hides the navbar while the page is scrolling downwards and shows it the other way. The plugin is able to show/hide the navbar programmatically as well.",
+  "author": {
+    "name": "István Ujj-Mészáros",
+    "url": "https://github.com/istvan-ujjmeszaros"
+  },
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/istvan-ujjmeszaros/bootstrap-autohidingnavbar.git"
+  },
+  "keywords": [
+    "jquery",
+    "plugin",
+    "bootstrap",
+    "ui",
+    "navigation"
+  ],
+  "homepage": "http://www.virtuosoft.eu/code/bootstrap-autohidingnavbar/",
```

Comment:
Let's use `https` https://www.virtuosoft.eu/code/bootstrap-autohidingnavbar/
Because when you access this web server use 80 port, it's always redirect to 443 port.
Also https is safe browsing.
Thanks.